### DISCUSSION TITLE
Not able to login

### DISCUSSION THREAD

**Discussion by pdiramerian** [community/community#125078] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

I created a new GitHub account but am unable to log in. I also tried the 'Forgot Password' feature, but it did not resolve the issue.

**Comment by ebndev** [comment]
Hi @pdiramerian,

This community is managed by GitHub employees, but it is not a support forum. The only place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them :)

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion. We appreciate your understanding!
